src/desktop/tabletinput.h

```diff
@@ -9,12 +9,44 @@ class QSettings;
 
 namespace tabletinput {
 
-void init(QApplication *app, const QSettings &cfg);
+#if defined(Q_OS_WIN)
+
+enum class Mode : int {
+	UNINITIALIZED,
```

Comment:
The style for enum members is `CamelCase` not `UPPER_SNAKE`?

Reply:
Oops, fixed.